src/Core/Utilities/ServiceContainer.cs

```diff
@@ -52,7 +52,13 @@ public static class ServiceContainer
                 organizationService);
             var vaultTimeoutService = new VaultTimeoutService(cryptoService, stateService, platformUtilsService,
                 folderService, cipherService, collectionService, searchService, messagingService, tokenService,
-                policyService, keyConnectorService, null, (extras) =>
+                policyService, keyConnectorService, 
+                (extras) =>
+                {
+                    messagingService.Send("locked", extras);
+                    return Task.FromResult(0);
```

Comment:
We can use `Task.CompletedTask` instead of `Task.FromResult(0)` which I think it's better to read and it's what's being used nowadays for tasks that have no await calls nor calls to other tasks.

Reply:
Fixed in #1830